kyaml/internal/forked/README.md

```diff
@@ -0,0 +1,11 @@
+# kyaml internal forks
+
+## qri-io/starlib
+
+This code is used by the starlark runtime. We copied it in to reduce the dependencies being brought over to kubectl by the kustomize integration. Should it need updating, do so via manual copy-paste.
+
+## go-yaml/yaml
+
+This code is used extensively by kyaml. It is a copy of upstream at a particular revision that kubectl is using, with [a change we need](https://github.com/go-yaml/yaml/pull/746) cherry-picked on top. For context on why the cherry-pick to an old revision is necessary, see https://github.com/kubernetes-sigs/kustomize/issues/3946.
```

Comment:
Note to self: update this last sentence if we use this--it's not to an old revision anymore